### ISSUE TITLE
Many feature requests, prioritized

### ISSUE THREAD

**Issue by 9z0** [orgzly-revived/orgzly-android-revived#88]
I was [involved](https://github.com/orgzly/orgzly-android/issues?q=is%3Aissue+involves%3A9z0) in feature requests in the original repo. Here they are **in order of importance and expected ease of implementation** and thank you for reviving Orgzly!

### High priority *or* easy to implement:

- orgzly/orgzly-android/issues/949

- orgzly/orgzly-android/issues/967

- orgzly/orgzly-android/issues/970

- orgzly/orgzly-android/issues/969

- orgzly/orgzly-android/issues/826

- orgzly/orgzly-android/issues/621

- [date picker ( "calendar view" ) **start with any day of the week**](orgzly/orgzly-android/issues/423)
----

### Medium priority:

- orgzly/orgzly-android/issues/985

- orgzly/orgzly-android/issues/978

- orgzly/orgzly-android/issues/972

- orgzly/orgzly-android/issues/971

- orgzly/orgzly-android/issues/968

- orgzly/orgzly-android/issues/129

- orgzly/orgzly-android/issues/828

- orgzly/orgzly-android/issues/146

- orgzly/orgzly-android/issues/824

- orgzly/orgzly-android/issues/823

- orgzly/orgzly-android/issues/820

- orgzly/orgzly-android/issues/821

- orgzly/orgzly-android/issues/819

- orgzly/orgzly-android/issues/312

----

### Low priority:

- orgzly/orgzly-android/issues/656

- orgzly/orgzly-android/issues/980

- orgzly/orgzly-android/issues/827

- orgzly/orgzly-android/issues/979

- orgzly/orgzly-android/issues/825

- orgzly/orgzly-android/issues/822

- orgzly/orgzly-android/issues/984

----

### Bug:
- orgzly/orgzly-android/issues/990

Thanks a lot!

**Comment by alensiljak** [comment]
I think 819 is already there in some form. Have you tried the current version's settings?

**Comment by 9z0** [comment]
> I think 819 is already there in some form. Have you tried the current version's settings?

Not yet. I am waiting a bit until potential early bugs are dealt with.

**Comment by alensiljak** [comment]
> Not yet. I am waiting a bit until potential early bugs are dealt with.

How will we know unless you try it out? :D

I've been using the 1.8.12 since it was published on F-Droid and have not experienced any issues. But my use cases may differ. In any case, the signature is different to Orgzly so you can have both installed.

**Comment by 9z0** [comment]
> How will we know unless you try it out? :D

You are right! But I am afraid of data-losing bugs that may go unnoticed by me even though I do regular backups.

I have added some issues to the first post for your consideration, prefixed by `(new)`.

**Comment by stefan2904** [comment]
- https://github.com/orgzly/orgzly-android/issues/353
is
- https://github.com/orgzly-revived/orgzly-android-revived/pull/37
or?

**Comment by 9z0** [comment]
> * [Sort contents of notebooks orgzly/orgzly-android#353](https://github.com/orgzly/orgzly-android/issues/353)
>       is
> 
>     * [Add menu option to sort lines in a note #37](https://github.com/orgzly-revived/orgzly-android-revived/pull/37)
>       or?

Not quiet.

The PR is about sorting ***contents*** of a note.

353 is about sorting notes themselves under the parent note/notebook.

**Comment by GfEW** [comment]
@9z0 

That's a great issue collection! Some of those are also in my own orgzly-revived reporting pipeline. Hope I get around to filing them, over time.

However, there are many more gems in your list. Do you consider recreating them here? I'm not overly confident it makes sense to discuss orgzly-revived issues in the orgzly repo.

**Comment by bughunter2** [comment]
The link for the 'first day of the week' feature seems to need fixing. The URL should be:
 - orgzly/orgzly-android/issues/423

I just voted for that feature. 👍

**Comment by 9z0** [comment]
> Do you consider recreating them here?

I might actually do that.
